cmd/zed/dev/indexfile/create/command.go

```diff
@@ -19,14 +18,21 @@ import (
 
 var Create = &charm.Spec{
 	Name:  "create",
-	Usage: "create [-f frameThresh] [ -o file ] -k field file",
-	Short: "create a key-only Zed index from a zng file",
+	Usage: "create [-f frametresh] [ -o file ] -k key[,key,...] file",
+	Short: "generate a Zed index file from one or more key-sorted Zed inputs",
 	Long: `
-The create command generates a key-only Zed index file comprising the values from the
-input taken from the field specified by -k.  The output index will have a base layer
-with search key called "key".
-If a key appears more than once, the last value in the input takes precedence.
-It is an error if the key fields are not of uniform type.`,
+The "zed indexfile create" command generates a Zed index from one or more Zed input files
+as a sectioned ZNG file with a layout desribed in the "zed indexfile" command.
+(Run "zed indexfile -h" for the description.)
+
+The bushiness of the B-Tree created is controlled by the framethresh parameter,
+which specifies a target size in bytes for each node in the B-Tree.
```

Comment:
As things stand, you have to read the code to know that the -f flag sets this parameter.
```suggestion
The bushiness of the B-tree created is controlled by the -f flag,
which specifies a target size in bytes for each node in the B-tree.
```